static/app/views/performance/queues/destinationSummary/messageSpanSamplesPanel.tsx

```diff
@@ -106,6 +143,24 @@ export function MessageSpanSamplesPanel() {
 
   const durationAxisMax = computeAxisMax([durationData?.[`avg(span.duration)`]]);
 
+  if (query.status.length > 0) {
```

Comment:
I would add comments as to why we are injecting these values here and on L150 so the next person reading this knows why it's there.

Reply:
thanks, added a comment and the function types

Reply:
And/or would be even better to name the variables appropriately. e.g.,

```jsx
const timeseriesFilters ...
const sampleFilters ...
```

then people are less likely to update the wrong variable